docs/ref/settings.txt

```diff
@@ -1251,6 +1251,20 @@ Subject-line prefix for email messages sent with ``django.core.mail.mail_admins`
 or ``django.core.mail.mail_managers``. You'll probably want to include the
 trailing space.
 
+.. setting:: EMAIL_USE_LOCALTIME
+
+``EMAIL_USE_LOCALTIME``
+-----------------------
+
+.. versionadded:: 1.11
+
+Default: ``False``
+
+Whether to send the SMTP date header in the local time zone (`True`) or in UTC
+(`False`). Using a value of `True` may be useful if your server is not set to
```

Comment:
Enabling this setting may be useful...
I think we could also chop "if your server is not set to the UTC time zone" to make the sentence simpler. The setting will still show the dates in the timezone of the server in that case... just that it won't have any noticeable effect, right?